Update monochrome android icon instructions (#10365)
godotengine/godot-docs: tutorials/export/exporting_for_android.rst

- **Main Icon:** The "classic" icon. This will be used on all Android versions up to Android 8 (Oreo), exclusive. Must be at least 192×192 px.
- **Adaptive Icons:** Starting from Android 8 (inclusive), `Adaptive Icons <https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive>`_ were introduced. Applications will need to include separate background and foreground icons to have a native look. The user's launcher application will control the icon's animation and masking. Must be at least 432×432 px.

> - **Themed Icons:** Starting from Android 13 (inclusive), Themed Icons were introduced. Applications will need to include a monochrome icon to enable this feature. The user's launcher application will control the icon's theme. Must be at least 432×432 px.

Reviewer: It's better to keep it with other icons. Just add optional here, e.g. Themed Icons (optional)

Author: The problem is that doesn't read well since the section starts with "There are two types of icons required by Godot:"

Reviewer: @skyace65 Then we can merge themed icon into Adaptive icon.

Adaptive Icons:
Adaptive icons were introduced in Android 8. Applications need to provide separate background and foreground icons to achieve a native look. Themed icons (optional), enabled by providing a monochrome icon, allow launchers to apply the user's chosen theme to the app icon from Android 13 onwards. The user's launcher application controls the icon's animation, masking, and theming based on user preferences. All icons must be at least 432x432 pixels.

.. seealso:: It's important to adhere to some rules when designing adaptive icons. `Google Design has provided a nice article <https://medium.com/google-design/designing-adaptive-icons-515af294c783>`_ that helps to understand those rules and some of the capabilities of adaptive icons.

.. caution:: The most important adaptive icon design rule is to have your icon critical elements inside the safe zone: a centered circle with a diameter of 66dp (264 pixels on ``xxxhdpi``) to avoid being clipped by the launcher.

> You can also optionally include a Themed Icon. Starting from Android 13 (inclusive), Themed Icons were introduced. Applications will need to include a monochrome icon to enable this feature. The user's launcher application will control the icon's theme. Must be at least 432×432 px.
> 

Reviewer: ```suggestion
```

> If you don't provide some of the requested icons, Godot will replace them using a fallback chain, trying the next in line when the current one fails:

Reviewer: ```suggestion
If you don't provide the requested icons (except for Monochrome), Godot will replace them using a fallback chain, trying the next in line when the current one fails:
```
